package.json

```diff
@@ -87,7 +87,9 @@
           "organizations-storage.addresses.collection.get",
           "organizations-storage.addresses.item.get",
           "organizations-storage.categories.collection.get",
+          "organizations-storage.organization-types.collection.get",
           "organizations-storage.categories.item.get",
+          "organizations-storage.organization-types.item.get",
```

Comment:
Looks like you use only `organizations-storage.organization-types.collection.get`